@here/harp-webpack-utils/scripts/HarpWebpackConfig.js

```diff
@@ -4,29 +4,45 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/*
```

Comment:
We shouldn't abandon TS.

It's easy to include `.*ts` in our webpack.configs, just like [here](https://github.com/heremaps/harp.gl/blob/master/webpack.tests.config.js#L112)

Reply:
Right, I only abandoned TS due to simplify CI / build issues. We now need to have the js for webpack-utils generated first before starting builds. Will look into using prepublish or similar.